zuul.d/projects.yaml

```diff
@@ -2,7 +2,9 @@
     check:
       jobs:
         - tox-linters
-        - ansible-role-tests
+        - ansible-role-tests-stable-py2
```

Comment:
The job definitions for ansible/ansible should be in ansible/ansible.
The reason I put them on ansible-zuul-jobs was to a hack to work out a network integration zuul role.

What you need to do is :

1. Install SF GH app on Ansible/ansible
2. Create .zuul.yml with job definitions on ansible/ansible
3. Put a depends-on on the ansible/ansible PR containing the .zuul.yml that depends on the roles on this ansible-zuul-jobs PR

Reply:
This PR is for testing `network-engine`, so I think this is the right place.

https://github.com/ansible-network/ansible-zuul-jobs/pull/13 will need moving to ansible/ansible

Reply:
ah sorry, got confused between the two links. This is good then